app/views/announcements/index.html.erb

```diff
@@ -1,6 +1,40 @@
 <p id="notice"><%= notice %></p>
 
-<h1>Announcements</h1>
+<h1 style="padding-left=20px">Announcements</h1>
+
+<div class="cardContainers">
+    <% @announcements.each do |announcement| %>
```

Comment:
I'm assuming that we still need to implement a pop up to view the comments/full information for the announcement, just making sure that is still the plan for a future commit.